pkg/operator/ceph/csi/cluster_config.go

```diff
@@ -103,6 +103,55 @@ func MonEndpoints(mons map[string]*cephclient.MonInfo, requireMsgr2 bool) []stri
 	return endpoints
 }
 
+// updateNetNamespaceFilePath modify the netNamespaceFilePath for all cluster IDs.
+// If holderEnabled is set to true. Otherwise, removes the netNamespaceFilePath value
+// for all the clusterIDs.
+func updateNetNamespaceFilePath(clusterNamespace string, cc csiClusterConfig) {
+	var (
+		cephFSNetNamespaceFilePath string
+		rbdNetNamespaceFilePath    string
+		nfsNetNamespaceFilePath    string
+	)
+
+	if IsHolderEnabled() {
+		for _, centry := range cc {
+			if centry.Namespace == clusterNamespace {
+				if centry.CephFS.NetNamespaceFilePath != "" {
+					cephFSNetNamespaceFilePath = centry.CephFS.NetNamespaceFilePath
+				}
+				if centry.RBD.NetNamespaceFilePath != "" {
+					rbdNetNamespaceFilePath = centry.RBD.NetNamespaceFilePath
+				}
+				if centry.NFS.NetNamespaceFilePath != "" {
+					nfsNetNamespaceFilePath = centry.NFS.NetNamespaceFilePath
+				}
+			}
+		}
+
+		for i, centry := range cc {
+			if centry.Namespace == clusterNamespace {
+				if centry.CephFS.NetNamespaceFilePath == "" {
+					cc[i].CephFS.NetNamespaceFilePath = cephFSNetNamespaceFilePath
+				}
+				if centry.RBD.NetNamespaceFilePath == "" {
+					cc[i].RBD.NetNamespaceFilePath = rbdNetNamespaceFilePath
+				}
+				if centry.NFS.NetNamespaceFilePath == "" {
+					cc[i].NFS.NetNamespaceFilePath = nfsNetNamespaceFilePath
+				}
```

Comment:
You dont need this check right? you can update even if the value is set and its no harm isn't it?

Reply:
yea, make sense.